libevm/VMSIMD.cpp

```diff
@@ -41,28 +41,30 @@ inline a32x8  const& v32x8 (u256 const& _stack_item) { return (a32x8&) *(a32x8*)
 inline a16x16 const& v16x16(u256 const& _stack_item) { return (a16x16&)*(a16x16*)&_stack_item; }
 inline a8x32  const& v8x32 (u256 const& _stack_item) { return (a8x32&) *(a8x32*) &_stack_item; }
 
+class Pow2Bits enum { BITS_8, BITS_16, BITS_32, BITS_64 };
```

Comment:
Thought I'd compiled it, but I did intend enum class, and I was pleasantly surprised not to see the problems you mentioned, so the config macro was probably turned off already.  A simple enum is what I'm sure compiled, and it makes sense in the scope of just one file.  I'll fix up the names.